Sofa/Component/Collision/Response/Contact/src/sofa/component/collision/response/contact/CollisionResponse.cpp

```diff
@@ -70,9 +72,9 @@ sofa::helper::OptionsGroup CollisionResponse::initializeResponseOptions(sofa::co
 void CollisionResponse::init()
 {
     Inherit1::init();
-    if (response.getValue().size() == 0)
+    if (d_response.getValue().size() == 0)
```

Comment:
```suggestion
    if (d_response.getValue().empty())
```

Reply:
when i compiled it showed this error:
error: no member named 'empty' in 'sofa::helper::OptionsGroup'
    if (d_response.getValue().empty())

Reply:
ok, leave it like that then